README.md

```diff
@@ -16,30 +16,61 @@
   * [Join the Gitter chat room](https://gitter.im/cocotb)
 
 ## Installation
+Cocotb can be installed by running `pip install cocotb`, and requires:
 
-Cocotb can be installed by running `pip install cocotb`.
+- Python 3.5+
+- A C++ compiler
+- An HDL simulator (such as [Icarus Verilog](http://iverilog.icarus.com/))
 
-## Quickstart
+For more details see [the documentation](https://cocotb.readthedocs.io/en/latest/quickstart.html#pre-requisites).
 
-    # Install pre-requisites (waveform viewer optional)
-    sudo yum install -y iverilog python-devel gtkwave
+## Usage
 
-    # Checkout git repositories
-    git clone https://github.com/cocotb/cocotb.git
+For a D flip-flop description:
 
-    # Install cocotb
-    pip install ./cocotb
+```verilog
+module dff (
+    output reg q,
+    input wire c, d,
+);
 
-    # Run the tests...
-    cd cocotb/examples/endian_swapper/tests
-    make
+always @(posedge clk) begin
+    q <= d;
+end
 
-    # View the waveform
-    gtkwave waveform.vcd
+endmodule
+```
 
+An example of a simple randomized cocotb testbench:
 
-## Tutorials and examples
+```python
+import random
+import cocotb
+from cocotb.clock import Clock
+from cocotb.triggers import FallingEdge
+
+@cocotb.test()
+async def test_simple_dff(dut):
+    """ Test that D propagates to Q """
+    
+    clock = Clock(dut.c, 10, units="us")  # Create a 10us period clock on port `c`
```

Comment:
c -> clk to match the RTL (and "c" isn't really a common name for a clock in RTL)